### PR TITLE
feat: support multiple data sources config

### PR DESCRIPTION
**Pull request by microud**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -15,7 +19,7 @@ declare module 'egg' {
 
   // extend your config
   interface EggAppConfig {
-    sequelize: sequelize.Options;
+    sequelize: sequelize.Options | DataSources;
```
Comment (dead-horse) [inline index.d.ts]: except sequelize opitons, `egg-sequelize` support `model`, `delegate`  options

Comment (microud) [reply index.d.ts]: 不是功能上的问题，只是不定义interface的话在用egg+typescript时配置多data sources会报错，可能我写的commit信息有偏差

Comment (dead-horse) [reply index.d.ts]: 我的意思是：除了 sequelize 自己支持的 options 之外，egg-sequelize 还支持 model / delegate 两个配置项，是否需要在这里也声明

**Comment by dead-horse** [comment]
4.3.0

**Pull request merged**